src/views/ConfusionMatrix/ConfusionMatrixViewData.ts

```diff
@@ -0,0 +1,34 @@
+import { validateObject } from "figurl"
+import { isArrayOf, isEqualTo, isNumber, isOneOf, isString } from "figurl/viewInterface/validateObject"
+
+export type ConfusionMatrixViewData = {
+    type: 'ConfusionMatrix'
+    sorting1UnitIds: (number | string)[]
+    sorting2UnitIds: (number | string)[]
+    unitEventCounts: {
+        unitId: number | string,
+        count: number
+    }[]
+    matchingUnitEventCounts: {
+        unitId1: number | string,
+        unitId2: number | string,
+        count: number
```

Comment:
So this would be the count of the number of events that match between unit id 1 and unit id 2 (from two different sortings), and we do the full pairwise comparison between the two sortings' units?

Reply:
yes